index.js

```diff
@@ -136,7 +141,34 @@ app.on('ready', () => {
 							window.setBounds({x: width - 1200, width: 1200, height: window.getSize()[1]});
 						}
 					},
-				}
+				}, {
+          label: 'Settings',
+          accelerator: 'CommandorControl+,',
```

Comment:
1. doesnt really need an accelerator right? its not something we expect people to use often
2. lets move it up and close to name it Change Quick Open Shortcut since it only does one thing right now?

Reply:
No, doesn't really need an accelerator. Will move it up and change the name. Will leave 2b to be tackled in a different PR for now.